.github/workflows/refapp-2x-service.yml

```diff
@@ -0,0 +1,45 @@
+name: RefApp 2.x Service
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+    paths:
+      - "qaframework-bdd-tests/src/**/*.*"
+      - "qaframework-bdd-tests/package.json"
```

Comment:
Add the path for the pom.xml that resides in qaframework-bdd-tests folder to have the test run on PR suggesting library upgrades

Reply:
Hello @kdaud  am still little confused about this change . Probably lets first handle your case and i will come back to this one as soon as it make sense to me.